.github/workflows/badgerdoc-build.yml

```diff
@@ -1,12 +1,12 @@
 name: badgerdoc build
 
 on:
-  push
+  pull_request
```

Comment:
What do you want to achieve with this event? pull_request will only be triggered on pull_request modifications (not related to new commits).

Reply:
> What is your objective with this event?

My objective is to reduce the number of runs, as this process is quite CPU and network intensive 

> pull_request will only be triggered on pull_request modifications (not related to new commits).

This job will be executed when a commit is added to a pull_request.